microsoft/botframework-solutions
# Fixes to .lu parsing scripts to use file location rather than current directory

## Description

Allows you to run the .bat files from any directory or as build events and see consistent behavior for the output.


## Testing Steps


1. Run the scripts from a directory other than the .bat file location and verify outputs are in the right place
2. Run one of the .bat files in a pre-build event and verify the build succeeds and the outputs are in the right place.
